internal/util/csiconfig.go

```diff
@@ -131,10 +131,14 @@ func CephFSSubvolumeGroup(pathToConfig, clusterID string) (string, error) {
 
 // GetMonsAndClusterID returns monitors and clusterID information read from
 // configfile.
-func GetMonsAndClusterID(options map[string]string) (string, string, error) {
-	clusterID, ok := options["clusterID"]
```

Comment:
why this check is moved out of `GetMonsAndClusterID` and done at multiple places before calling this function. IMO this is error-prone and the user needs to be aware that he/she has to do this check before calling `GetMonsAndClusterID` and it also increases the code duplication and code complexity as one more if check is needed for the caller function.

Reply:
On another note,  we unnecessarily have to create a options map when using this in other places which also leads to code duplication. 

When we have the parameter as clusterID string, it is more obvious to have cluster ID exists check if it's being read from a map. 
@Madhu-1 ^^

Reply:
in that case, please move the validation to a new helper function or define const of `"clusterID"` map key

Reply:
I think one thing we can do here is this:

createFunctions to do one thing:

```
 GetMons(clusterID,..)
  just fetch the mons
GetClusterID(options,..)
 just validate the clusterID ..
Check/GetClusterIDMapping()??
```

Reply:
@humblec @Madhu-1  PTAL and let me know if these changes are fine. :)